### PR TITLE
release 0.9

### PR DESCRIPTION
**Pull request by quasiben**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by quasiben** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+{% set pkg_loc = "py2.py3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 44ba7cd2e2ca6004f8cbcc358816fea68a898da509cd41dd85e7c4a7c7fcb4ed
+  url: https://files.pythonhosted.org/packages/{{ pkg_loc }}/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-{{ pkg_loc }}-none-any.whl
```
Comment (jakirkham) [inline recipe/meta.yaml]: There should be an `sdist` on PyPI. They also included a `sha256` checksum for it. Would recommend using the `sdist` instead (unless there is a reason not to)

If the templated URL is out-of-date here, would look at [this example in `staged-recipes`]( https://github.com/conda-forge/staged-recipes/blob/7033c8924612d09c90e60ed6eda6f2986414d94b/recipes/example/meta.yaml#L14 )

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
 
 build:
   number: 0
   skip: true  # [win or py<35]
   entry_points:
     - dask-yarn=dask_yarn.cli:main
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script:
+    - python -m pip install --no-deps --ignore-installed {{ name }}-{{ version }}-{{ pkg_loc }}-none-any.whl
+    - cp {{ SP_DIR }}/{{ name }}-{{ version }}.dist-info/LICENSE.txt {{ SRC_DIR }}/LICENSE.txt
```
Comment (jakirkham) [inline recipe/meta.yaml]: Think we can revert this if the `sdist` works. The license file is already included there and conda-build knows how to pick that up

Comment (jakirkham) [reply recipe/meta.yaml]: Looks like this reappeared, which is causing the CI issue

Comment (jakirkham) [reply recipe/meta.yaml]: Fixed in latest commit

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

**Comment by jakirkham** [comment]
LGTM. Thanks Ben! 😄

@mrocklin, could you please review? 🙂

**Comment by quasiben** [comment]
@mrocklin the review here might just be you hitting the green merge button

**Pull request merged**